application/config/automatic_config.go

```diff
@@ -61,7 +62,12 @@ func (c *Config) normalizePath(foundPath string) (string, bool) {
 	return path, false
 }
 
-func (c *Config) determineMavenHome() {
+func (c *Config) mavenDefaults() {
+	// explicitly and always use headless mode
+	mavenOptsVarName := "MAVEN_OPTS"
+	mavenOpts := fmt.Sprintf("%s %s", os.Getenv(mavenOptsVarName), "-Djava.awt.headless=true")
+	_ = os.Setenv(mavenOptsVarName, mavenOpts)
+
```

Comment:
This is unrelated to the actual PR goal, but the JVM was popping up UI and stealing focus - we don't want that in LS.